BrainCloudClient/tests/TestLeaderboard.cs

```diff
@@ -334,6 +335,26 @@ public void TestPostScoreToDynamicLeaderboardNullRotationTime()
 
             tr.Run();
         }
+        
+        [Test]
+        public void TestPostScoreToDynamicGroupLeaderboardDaysUTC()
+        {
+            TestResult tr = new TestResult(_bc);
+
+            _bc.LeaderboardService.PostScoreToDynamicGroupLeaderboardDaysUTC(
+                _dynamicLeaderboardId + "_" + BrainCloudSocialLeaderboard.SocialLeaderboardType.LAST_VALUE.ToString() + "_" + _random.Next(),
+                _groupLeaderboardId,
+                100,
+                Helpers.CreateJsonPair("testDataKey", 400),
+                BrainCloudSocialLeaderboard.SocialLeaderboardType.HIGH_VALUE,
+                (ulong)TimeUtil.UTCDateTimeToUTCMillis(TimeUtil.LocalTimeToUTCTime(System.DateTime.Now.AddDays(5))),
```

Comment:
Don't add 5 days to now. This is the date to start the rotation. should just be now.